ush/global_ens/evs_global_ens_atmos_grid2obs.sh

```diff
@@ -95,6 +132,9 @@ for field in $fields ; do
        fi
     fi
     for metplus_job in $metplus_jobs ; do
+      ****************************************
```

Comment:
@BinbinZhou-NOAA Also, on line 135 in [ush/global_ens/evs_global_ens_atmos_grid2obs.sh](https://github.com/NOAA-EMC/EVS/pull/357/files#diff-cebabbc3f476e70043bbb903a0e9249f02eacce0f9ab75f9c36afbe2d82479a0), please add a # before the line of ************** so that the code knows it's a comment. Thanks!

CC @ShelleyMelchior-NOAA @PerryShafran-NOAA